packages/run-protocol/test/swingsetTests/setup.js

```diff
@@ -202,19 +203,23 @@ const buildOwner = async (
   /** @type {AssetReservePublicFacet} */
   // @ts-expect-error cast, never used
   const reservePublicFacet = null;
+  /** @type {Amount} */
+  // @ts-expect-error cast, never used
```

Comment:
the comment "tests are currently disabled" was confusing to me at first: what tests.

in 8f3af1bdd I replaced using the other comment. as I write this I suspect it's not as true in this case, but either the whole suite is going away or when it's reinstated this comment won't matter much